activemq_xml/datadog_checks/activemq_xml/data/conf.yaml.example

```diff
@@ -250,10 +250,6 @@ instances:
     ## @param tls_ignore_warning - boolean - optional - default: false
     ## If `tls_verify` is disabled, security warnings are logged by the check.
     ## Disable those by setting `tls_ignore_warning` to true.
-    ##
-    ## Note: `tls_ignore_warning` set to true is currently only reliable if used by one instance of one integration. 
-    ## If enabled for multiple instances, spurious warnings might still appear even if `tls_ignore_warning` is set
-    ## to true.
```

Comment:
Looks like your base may not be the latest on master as it's removing the note added here https://github.com/DataDog/integrations-core/blob/e2282c563665ca197e742500ec7f547b97ccdf23/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/http.yaml#L194-L196